presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/TestStaticSelector.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.resourceGroups;
+
+import com.facebook.presto.spi.resourceGroups.ResourceGroupId;
+import com.facebook.presto.spi.resourceGroups.SelectionContext;
+import com.google.common.collect.ImmutableSet;
+import org.testng.annotations.Test;
+
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestStaticSelector
+{
+    @Test
+    public void testStaticSelectorUserRegex()
```

Comment:
The tests can be simplified a bit if you define the below helper. [Here](https://gist.github.com/nezihyigitbasi/70730cac92d8b239a603155cb337c0c4) is the updated version.

```
private SelectionContext newSelectionContext(String user, String source, Set<String> tags)
{
    return new SelectionContext(true, user, Optional.ofNullable(source), tags, 1, Optional.empty());
}
```